frontend/app/(footer)/profile/page.tsx

```diff
@@ -158,9 +158,11 @@ const Profile = () => {
 
     if (files && files.length > 0) {
       const file = files[0];
+      const originalFileName = file.name;
+      const modifiedFileName = originalFileName.replace(/[(){}[\]]/g, "");
```

Comment:
it seems all file names pertaining to profile image should undergo this transformation. We should guarantee that we enforce this transformation somehow every single time. Doing the transformation as an individual step makes it likely that we will somehow forget to do this transformation. We should abstract all these operations to another function which handles these transformations.

Reply:
redone as uuid